specs/tendrl_host_listing_monitoring_summary_api.adoc

```diff
@@ -0,0 +1,187 @@
+// vim: tw=79
+
+= Host Listing Backend Support
+
+== Introduction
+
+The host listing page displays the following details per host:
+
+* Hostname and status
+* Storage used, total and percentage used
+* Cpu percentage used
+* Memory used, total and percentage used
+* Name of Cluster to which host belongs
+* Role of the node in the cluster
```

Comment:
This spec is only meant to deal with the monitoring aspect of details required by host listing page in UI like the memory, cpu, storage utilizations and alert counter only. The rest of the details that feature in "Introduction" will be exposed by tendrl-api. I think @anivargi might have a spec to cover those details. @anivargi could you please comment.